packages/grafana-ui/src/components/Table/ImageCell.tsx

```diff
@@ -30,7 +30,13 @@ export const ImageCell: FC<TableCellProps> = (props) => {
     <div {...cellProps} className={tableStyles.cellContainer}>
       {!link && <img src={displayValue.text} className={tableStyles.imageCell} />}
       {link && (
-        <a href={link.href} onClick={onClick} target={link.target} title={link.title} className={tableStyles.cellLink}>
+        <a
+          href={link.href}
+          onClick={onClick}
```

Comment:
Interesting, do you know why we're using both `href` and `onClick` handler here?

Reply:
No, I don't.

Reply:
would love to know more about why we're using both

Reply:
Aha, ok I guess it handles the case when `link.onClick` doesn't exist but there is a `link.href`. Hmm, looks kind of messy.

Reply:
yeah